src/Sylius/Bundle/CoreBundle/Processor/RequestProductVariantCatalogPromotionRecalculate.php

```diff
@@ -16,15 +16,15 @@
 use Sylius\Bundle\CoreBundle\CommandDispatcher\ApplyCatalogPromotionsOnVariantsCommandDispatcherInterface;
 use Sylius\Component\Core\Repository\ProductVariantRepositoryInterface;
 
-final class AllCatalogPromotionsProcessor implements AllCatalogPromotionsProcessorInterface
+final class RequestProductVariantCatalogPromotionRecalculate implements RequestProductVariantCatalogPromotionRecalculateInterface
```

Comment:
Why did you refactor the name of this service? To be honest, I'm not a fan of the new one 🤔

Reply:
To be consistent with other processors from this namespace, maybe sth like:
```suggestion
final class AllProductVariantsCatalogPromotionsProcessor implements AllProductVariantsCatalogPromotionsProcessorInterface
```